backend/src/apiserver/common/config.go

```diff
@@ -23,7 +23,9 @@ import (
 )
 
 const (
-	MultiUserMode     string = "MULTIUSER"
+	MultiUserMode       string = "MULTIUSER"
+	podNamespace        string = "POD_NAMESPACE"
+	podNamespaceDefault string = "kubeflow"
```

Comment:
a quick question: the standalone deployment using "kubeflow" as default, the hosted use "kubeflow" or "default"? Does it matter in this case?

Reply:
It doesn't matter. The default branch will only trigger in tests.

In k8s env, POD_NAMESPACE environment variable is always provided to each pod, so api server will get the correct value injected.

Reply:
I see.
/lgtm

Reply:
Hold on, actually that's not set by k8s, but by us: https://github.com/kubeflow/pipelines/search?q=POD_NAMESPACE&unscoped_q=POD_NAMESPACE

So I guess it's better to report an Error if it's not provided.

Reply:
I've changed to not default to kubeflow and init tests with POD_NAMESPACE env var